templates/prusa-mini-printer.template.cfg

```diff
@@ -0,0 +1,254 @@
+variable_preheat_extruder: True
+# Calibrate the bed mesh in the START_PRINT macro.
+# Set to false to skip BED_MESH_CALIBRATE, it will still load the BED_MESH 
+# with the name "ratos", be sure to save your bed_mesh profile with that name.
+# or override the _START_PRINT_BED_MESH macro to implement your own mesh handling logic.
+variable_calibrate_bed_mesh: True
+# Print a prime line at the end of the START_PRINT macro
+# set to False to disable nozzle_priming.
+variable_nozzle_priming: "primeline"
+# Park in the back when waiting for the extruder to heat up
+# set to "front" to park in the front, or "center" to park in the center.
+variable_start_print_park_in: "front"
+# Height to park it when waiting for extruder to heat.
+variable_start_print_park_z_height: 50
+# Skew profile to load before starting the print 
+# uncomment this to use your calibrated skew correction profile.
+variable_skew_profile: "my_skew_profile"
+# Park in the back after the print has ended or was cancelled.
+# set to "front" to park in the front, or "center" to park in the center.
+variable_end_print_park_in: "back"
+# Park in the back when the print is paused.
+# set to "front" to park in the front, or "center" to park in the center.
+variable_pause_print_park_in: "front"
+# Set the speed for travel moves in RatOS Macros in mm/s.
+variable_macro_travel_speed: 165
+# Set the length and speed for the unload filament
+variable_filament_unload_length: 70
+variable_filament_unload_speed: 5
+# Set the length and speed for the load filament
+variable_filament_load_length: 100
+variable_filament_load_speed: 10
+#############################################################################################################
+### PRINTER CONFIGURATION
+### Customize the defaults to your specific build
+#############################################################################################################
+
+[mcu]
+serial: /dev/serial/by-id/usb-Klipper_stm32f407xx_2C003B000B47393339323834-if00
+
+[tmc2209 extruder]
+diag_pin: ^e_diag_pin
+driver_SGTHRS: 100
+sense_resistor: 0.22
+
+[extruder]
+pressure_advance: 0.04 #this is a reference value, should be calibrated on each printer
+control = pid
+pid_kp = 20.184
+pid_ki = 0.836
+pid_kd = 121.862
+
+[heater_bed]
+control = pid
+pid_kp = 63.567
+pid_ki = 1.084
+pid_kd = 932.049
+
+[respond]
+default_type: command
+
+[virtual_sdcard]
+path:/home/pi/printer_data/gcodes
+
+[pause_resume]
+recover_velocity: 50.
+
+[exclude_object]
+
+[stepper_z]
+position_max: 185
+
+[probe]
+z_offset = 2.0
+
+# Home, get position, throw around toolhead, home again.
+# If MCU stepper positions (first line in GET_POSITION) are greater than a full step different (your number of microsteps), then skipping occured.
+# We only measure to a full step to accomodate for endstop variance.
+# Example: TEST_SPEED SPEED=300 ACCEL=5000 ITERATIONS=10
+
+#[mcu USB-ADXL345]
+#serial: /dev/serial/by-id/usb-Klipper_stm32f103xe_36FFD7054858383038652443-if00
```

Comment:
Remove this, it's not a thing in RatOS. We could add separate support for a USB adxl345 board if necessary.

Reply:
In the case of the prusa mini with the buddy board you must connect the ADLX345 sensor to the rpi, there is no possibility to connect to the board. So I added a new toolboard input-shaper-usb.cfg, the problem is that the user should override the pinout (maybe) and the mcu serial

Reply:
You can connect it to the pi and use the rpi config: https://github.com/Rat-OS/RatOS-configuration/blob/v2.x/sensors/rpi-adxl345.cfg and https://github.com/Rat-OS/RatOS-configuration/blob/v2.x/boards/rpi/config.cfg. A usb controller used solely for ADXL345 is not how you connect an ADXL345 to the pi.

You'll want to have a config that sets
```
[resonance_tester]
accel_chip: adxl345 rpi
```

Reply:
I understand, so I have left the same configuration than for the v-minion